localtools/src/integrationTest/kotlin/testconfiguration/services/InvoiceOnboardingService.kt

```diff
@@ -0,0 +1,161 @@
+package testconfiguration.services
+
+import cosmos.tx.v1beta1.ServiceOuterClass.BroadcastMode
+import io.provenance.classification.asset.client.client.base.ACClient
+import io.provenance.classification.asset.client.domain.execute.OnboardAssetExecute
+import io.provenance.classification.asset.client.domain.model.AccessRoute
+import io.provenance.classification.asset.client.domain.model.AssetIdentifier
+import io.provenance.classification.asset.util.extensions.wrapListAc
+import io.provenance.classification.asset.util.wallet.ProvenanceAccountDetail
+import io.provenance.client.grpc.BaseReqSigner
+import io.provenance.client.protobuf.extensions.toAny
+import io.provenance.client.protobuf.extensions.toTxBody
+import io.provenance.metadata.v1.MsgWriteRecordRequest
+import io.provenance.metadata.v1.MsgWriteScopeRequest
+import io.provenance.metadata.v1.MsgWriteSessionRequest
+import io.provenance.metadata.v1.Party
+import io.provenance.metadata.v1.PartyType
+import io.provenance.metadata.v1.RecordInput
+import io.provenance.metadata.v1.RecordInputStatus
+import io.provenance.metadata.v1.RecordOutput
+import io.provenance.metadata.v1.ResultStatus
+import io.provenance.scope.encryption.crypto.Pen
+import io.provenance.scope.encryption.ecies.ProvenanceKeyGenerator
+import io.provenance.scope.objectstore.client.OsClient
+import io.provenance.scope.util.MetadataAddress
+import io.provenance.scope.util.toByteString
+import io.provenance.spec.AssetSpecifications
+import mu.KLogging
+import testconfiguration.extensions.getContractSpecFromScopeSpec
+import testconfiguration.extensions.toBase64StringAc
+import testconfiguration.models.TestAsset
+import testconfiguration.util.AppResources
+import java.util.UUID
+import java.util.concurrent.TimeUnit
+
+class InvoiceOnboardingService(
+    private val acClient: ACClient,
+    private val osClient: OsClient,
+) {
+    private companion object : KLogging()
+
+    fun onboardTestAsset(
```

Comment:
EZPZ onboard an asset.  It just loads up object store with total garbage JSON, but it works lol